lib/game.rb

```diff
@@ -1,22 +1,35 @@
 require './required_files'
 
 class Game
+  attr_reader :game_id,
+              :season,
+              :type,
+              :date_time,
+              :away_team_id,
+              :home_team_id,
+              :away_goals,
+              :home_goals,
+              :venue,
+              :venue_link
 
-  attr_reader :game_id, :season, :type, :date_time, :away_team_id, :home_team_id,
-  :away_goals, :home_goals, :venue, :venue_link
-
-  def initialize(game_id, season, type, date_time, away_team_id, home_team_id,
-    away_goals, home_goals, venue, venue_link)
+  def initialize(row)
```

Comment:
Nice job refactoring this Rue. I merged it on my local machine, and I am getting a few tests that don't pass in the game_spec, game_team_spec, and team_spec. I think since the initialize method only takes one argument now there is a problem when the tests are creating the game objects because the tests give a bunch of arguments.

Reply:
Changes made to game_spec, team_spec, game_team_spec as mentioned! rspec is now passing for all files.